src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -21867,18 +21867,15 @@ void M(Delegate d, string? s)
     {
         if (s != string.Empty) s.ToString(); // warn
         d.BeginInvoke(s, null, null);
-        s.ToString(); // warn 2
+        s.ToString();
```

Comment:
I think removing this warning makes sense because in the case where a null `s` is passed to the method, we've already hit the first call to `s.ToString()`.